portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,33 +15,66 @@
 package com.google.sps.servlets;
 
 import com.google.gson.Gson;
+import com.google.sps.data.Comment;
+import sun.tools.jconsole.JConsole;
 
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-import java.util.Arrays;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
 import java.util.List;
 
 /** Servlet that returns some example content.*/
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private List<String> messages = Arrays.asList("Brilliant!", "Pretty!", "Great!");
+  private List<Comment> comments;
+
+  @Override
+  public void init() {
+    comments = new ArrayList<>();
+  }
   
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String json = convertToJsonUsingGson(this.messages);
+    String json = convertToJsonUsingGson(this.comments);
 
     // Send the JSON as the response
     response.setContentType("application/json;");
     response.getWriter().println(json);
   }
 
-  private String convertToJsonUsingGson(List<String> messages) {
+  private String convertToJsonUsingGson(List<Comment> comments) {
     Gson gson = new Gson();
-    String json = gson.toJson(messages);
+    String json = gson.toJson(comments);
     return json;
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get comments from the form
+    String name = getParameter(request, "name", "");
+    String comment = getParameter(request, "comment", "No comments");
+
+    // Respond with the result.
+    comments.add(new Comment(name, comment, LocalDateTime.now()));
+
+    // Redirect back to the HTML page.
+    response.sendRedirect("/index.html");
+  }
+
+  /**
+   * @return the request parameter, or the default value if the parameter
+   *         was not specified by the client
```

Comment:
Is there some thing wrong with the formatting? It's showing as broken lines on my github page.

Reply:
May i have the screenshot to see how it is shown in ur github?